pkg/build/controller/config_controller.go

```diff
@@ -55,9 +55,15 @@ func (c *BuildConfigController) HandleBuildConfig(bc *buildapi.BuildConfig) erro
 	}
 
 	glog.V(4).Infof("Running build for BuildConfig %s/%s", bc.Namespace, bc.Name)
+
+	buildTriggerCauses := []buildapi.BuildTriggerCause{}
 	// instantiate new build
 	lastVersion := 0
 	request := &buildapi.BuildRequest{
+		TriggeredBy: append(buildTriggerCauses,
+			buildapi.BuildTriggerCause{
+				Reason: "Triggered by config change",
```

Comment:
@bparees this reads weird... `TriggeredBy: "Triggered by config change`. I think it should be just `TriggeredBy: "Build configuration change"`